packages/module-auto-icons/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@wxt-dev/module-auto-icons",
+  "version": "1.0.0",
+  "type": "module",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      }
```

v0.19.0 is queued up and about to be released, the breaking change being dropping CJS support. So you can remove the "require". I should remove it from the other modules as well...

```suggestion
      "types": "./dist/index.d.mts",
      "default": "./dist/index.mjs"
```

---

Added in my commit